### DISCUSSION TITLE
Error while trying to select only certain fields with findMany()

### DISCUSSION THREAD

**Discussion by DavidRodarte** [prisma/prisma#12091] [Q&A] [answered]
I'm trying to use the select property from findMany in order to fetch only certain fields from my model called Ticket 

This is my model
<img width="761" alt="image" src="https://user-images.githubusercontent.com/23324788/156200411-11188591-c981-4a22-9dbf-d29f445da4bd.png">

But I'm getting this error saying I'm missing some properties, but in this specific case I only want to get my "uuid" field.
<img width="785" alt="image" src="https://user-images.githubusercontent.com/23324788/156200658-9eea54f4-0f30-4a63-8e46-7f57b4bff7e7.png">

Any idea on how to solve this issue? 

I'm using:
Prisma v3.10.0
NestJs  v8.2.1

**Comment by austincrim** [comment, marked as answer]
Hey @DavidRodarte 👋 ,

You have defined your `getTickets` function to return `Ticket[]`, but since you are only selecting `uuid` in your query, the return type will only include the `uuid` field as well.

If you change your function signature, it should fix the problem.

```ts
async getTickets(): Promise<{uuid: string}[]> {
  ...
}
```

Let me know if that works!

**Reply by DavidRodarte** [reply]
Thank's, I completely forgot to change the return type

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
